tests/Tests/500__jupyterhub/test-folder-permissions.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Documentation       Test Suite to verify installed library versions
+
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             JupyterLibrary
+
+Suite Setup         Load Spawner Page
+Suite Teardown      End Web Test
+
+
+*** Test Cases ***
+Verify Folder Permissions
+    [Documentation]    It checks Access, Gid, Uid of /opt/app-root/lib and /opt/app-root/shares
+    [Tags]    ODS-486
+    ...       Tier2
+    @{list_of_images} =    Create List
+    Append To List
+    ...    ${list_of_images}
+    ...    s2i-minimal-notebook
+    ...    s2i-generic-data-science-notebook
+    ...    pytorch
+    ...    tensorflow
+    ${permissions} =    Create List    0775    0    1001
```

Comment:
${permissions} and ${path} can also be in the ***Variables*** section. 
 
 Consider renaming them to ${expected_permissions} and ${folders_to_check}

Reply:
Done